packages/client/src/interface/navigation/servers/UserMenu.tsx

```diff
@@ -37,6 +37,16 @@ interface Props {
   anchor: Accessor<HTMLDivElement | undefined>;
 }
 
+const TruncatedStatusText = styled("div", {
+  base:
+  {
+    maxWidth: '300px',
```

Comment:
Little bit arbitrary, but looks fine to me.

Reply:
Should I use a var that you guys approve of?